lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -3966,12 +4052,17 @@ void StmtEmitter::collectNamesEmitDecls(Block &block) {
   if (valuesToEmit.empty())
     return;
 
-  size_t maxDeclNameWidth = collector.getMaxDeclNameWidth();
-  size_t maxTypeWidth = collector.getMaxTypeWidth();
+  // Record maxDeclNameWidth and maxTypeWidth in the current scope.
+  maxDeclNameWidth = collector.getMaxDeclNameWidth();
+  maxTypeWidth = collector.getMaxTypeWidth();
 
   if (maxTypeWidth > 0) // add a space if any type exists
     maxTypeWidth += 1;
 
+  // In the new emission mode, we don't do forward declarations.
```

Comment:
Can you also add something like TODO: once this is the only emission mode, remove the below? It'd be nice if we could just search all the TODOs in this file at a later date.